test/integration/test-cases/happypath.sh

```diff
@@ -13,33 +13,39 @@ info "Diffing ${dir}/happypath.yaml with altered output"
 bash -c "${helmfile} -f ${dir}/happypath.yaml diff --output simple --detailed-exitcode; code="'$?'"; [ "'${code}'" -eq 2 ]" || fail "unexpected exit code returned by helmfile diff"
 
 info "Templating ${dir}/happypath.yaml"
-rm -rf ${dir}/tmp
-${helmfile} -f ${dir}/happypath.yaml --debug template --output-dir tmp
+pushd ${dir}
```

Comment:
I think this isn't going to work as now you can't find `./diff-yamls` invoked at line 27 within the current workdir.
Also, the best practice would be to keep the set of changes made in PR to be tightly scoped, so that we don't need many back-and-forths 😉